<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MandelbrotConfigurationDialog</class>
 <widget class="QDialog" name="MandelbrotConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MandelJuliaConfigurationDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Domain</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>315</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>X min</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="mySpinXMin">
         <property name="decimals">
          <number>15</number>
         </property>
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>-2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>X max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="mySpinXMax">
           <property name="decimals">
            <number>15</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Y min</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="mySpinYMin">
         <property name="decimals">
          <number>15</number>
         </property>
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>-2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Y max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="mySpinYMax">
           <property name="decimals">
            <number>15</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>341</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Image size</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Size X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mySpinSizeX">
       <property name="maximum">
        <number>20000</number>
       </property>
       <property name="value">
        <number>800</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Size Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mySpinSizeY">
       <property name="maximum">
        <number>20000</number>
       </property>
       <property name="value">
        <number>800</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>General</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Iterations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mySpinIterations">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>133</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QCheckBox" name="myCheckBoxLogLog">
       <property name="text">
        <string>Log log smoothing</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QDoubleSpinBox" name="mySpinBailout">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100000000000000000000.000000000000000</double>
    </property>
    <property name="value">
     <double>128.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bailout</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>195</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="myButtonOk">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="myButtonCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>341</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Renderer</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Renderer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="myComboRenderer">
         <item>
          <property name="text">
           <string>Black &amp; white</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Random</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Random 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sine</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="myButtonConfigureRenderer">
         <property name="text">
          <string>Configure</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>160</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QCheckBox" name="myCheckBoxUseFunction">
      <property name="text">
       <string>Function</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="myEditFunction"/>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="myComboComputationType">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>610</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Normal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Triangle inequality</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Average curvature 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Average curvature 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Average curvature 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stripe average</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fold line</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fold square</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fold circle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Exp smoothing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Z smoothing</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>670</y>
     <width>197</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QCheckBox" name="myCheckBoxUseAddEnd">
      <property name="text">
       <string>Function (addEnd, branching)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="myEditAddEndFunction"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="myCheckBoxLambertCorrection">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>560</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lambert correction</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>740</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QCheckBox" name="myCheckBoxUseRerange">
      <property name="text">
       <string>Re-range value function </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="myEditRerangeFunction"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>800</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <widget class="QCheckBox" name="myCheckBoxUseRerangeHeight">
      <property name="text">
       <string>Re-range 3D height</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="myEditRerangeHeight"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
